tests/console/sudo.pm

```diff
@@ -107,6 +107,16 @@ sub run {
     # Check if sudo asks for the root password.
     # On Azure from SLE15 onwards, 'Defaults targetpw' is disabled. There sudo is expected to ask for the user password
     my $exp_user = (is_azure && is_sle(">=15")) ? "$testapi::username" : "root";
+    # Workaround for the the 15-SP2 images, where the change is not yet applied
+    # 15-SP2 will get this change eventually, but it is unknown when the images will be refreshed.
+    if (is_azure && is_sle("=15-SP2")) {
+        select_console 'root-console';
+        if (script_output("cat /etc/sudoers") =~ m/^Defaults targetpw/m) {
```

Comment:
```suggestion
        if (script_output("sudo cat /etc/sudoers") =~ m/^Defaults targetpw/m) {
```
can't we just use sudo here instead of switching consoles ? :thinking:

Reply:
Because `sudo` requires a password, see https://openqa.suse.de/tests/13942279#step/sudo/14.

Reply:
ah , right !